Dockerfile

```diff
@@ -4,8 +4,16 @@ USER node
 
 WORKDIR /home/node
 
-COPY --chown=node:node . .
+COPY --chown=node:node ./package.json .
 
 RUN npm --silent i
 
-CMD [ "node", "main.js" ]
+USER root
+
+RUN apk update && \
```

Comment:
this should be done before https://github.com/fulcanelly/auclean-bot/blob/490b171c87bc2e2376d0b793a2b9d2fd4fc33503/Dockerfile#L3